internify/src/actions/userTypes.js

```diff
@@ -0,0 +1,4 @@
+export const ADD_USER = 'ADD_USER';
+export const DELETE_USER = 'DELETE_USER';
+export const DELETE_ALL_USERS = 'DELETE_ALL_USERS';
+export const SEND_SELECTED_USER = 'SEND_SELECTED_USER';
```

Comment:
These seem like actions for `User` rather than `UserType`. Should we rename this file to something like `userActions`?